srcs/Utils.cpp

```diff
@@ -399,4 +399,22 @@ getMethodType(std::string str)
 	return (DEFAULT);
 }
 
+bool
+checkValidFileExtension(std::string file_name, std::vector<std::string> ext_list)
+{
+	std::vector<std::string> tmp;
+	std::string extension;
+
+	if (file_name.find(".") == std::string::npos)
+		return (false);
+	tmp = ft::split(file_name, '.');
+	extension = tmp.back();
+	for (int i = 0 ; i < ext_list.size() ; i++)
```

Comment:
```suggestion
	for (size_t i = 0 ; i < ext_list.size() ; i++)
```

> That's it. On some architecture and for some compilers it might be the same as size_t, in some others it might not. The assumption that a variable of type size_t can hold the same values than one of type std::vector<T>::size_type can fail.

- https://stackoverflow.com/questions/14599057/how-compare-vector-size-with-an-integer